compiler/passes/normalize.cpp

```diff
@@ -1744,6 +1764,14 @@ static void change_method_into_constructor(FnSymbol* fn) {
     }
   }
 
+  if (ct->instantiationStyle == DEFINES_INITIALIZER) {
+    ArgSymbol* meme = new ArgSymbol(INTENT_BLANK, "meme", ct, NULL,
```

Comment:
Though scopeResolve is capable of distinguishing the cases, I encountered an error for failing to pass in an lvalue when I tried to make similar changes to what was suggested here.  The changes suggested to this file did not cause new failure modes among my initializers tests and futures, but I'm suspicious given how many initializers tests aren't in a position to work yet.  I think for now I will keep these strategies in sync and leave the arg intent the way it is since this is the strategy that has worked with the default wrapper passing in the memory, but will keep in mind that we will likely want to use those intents in the future (and that it's suspicious that we don't and yet things still work)